src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -922,8 +1040,7 @@ export class BundleDataClient {
                     // sent for this fill and refund them to the filler, because this value would not be paid out
                     // otherwise. These deposits can no longer expire and get refunded as an expired deposit,
                     // and they won't trigger a pre-fill refund because the fill is in this bundle.
-                    // Pre-fill refunds only happen when deposits are sent in this bundle and the
-                    // fill is from a prior bundle. Paying out the filler keeps the behavior consistent for how
```

Comment:
Why is this comment dropped btw?

Reply:
Holdover changes from the WIP - thanks for pointing out - removed +1